test/elixir/test/replication_test.exs

```diff
@@ -9,9 +9,9 @@ defmodule ReplicationTest do
   # TODO: Parameterize these
   @admin_account "adm:pass"
   @db_pairs_prefixes [
-    {"local-to-local", "", ""},
-    {"remote-to-local", "http://127.0.0.1:15984/", ""},
-    {"local-to-remote", "", "http://127.0.0.1:15984/"},
+    # {"local-to-local", "", ""},
+    # {"remote-to-local", "http://127.0.0.1:15984/", ""},
+    # {"local-to-remote", "", "http://127.0.0.1:15984/"},
```

Comment:
Is there reason why above 3 pairs are disabled for now?  Do we have flaky failures for these?

Reply:
oops that was for testing. I'll add them back